src/main/java/com/example/demo/response/dto/UserDto.java

```diff
@@ -0,0 +1,95 @@
+package com.example.demo.response.dto;
+
+import java.io.Serializable;
+import java.util.Optional;
+
+import org.springframework.stereotype.Component;
+
+import com.example.demo.model.Address;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@Component
+@SuppressWarnings("serial")
+public class UserDto implements Serializable{
+	
+	private String name;
+	
+	@JsonIgnoreProperties("user")
+	private Optional<Address> address;
```

Comment:
Don't use optional in DTO use directly AddressDTO which can either be null or an object and also create AddressDto rather than using address model class directly